frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -112,6 +129,22 @@ const loadPriorityMap: Record<TileId, number> = {
     [TileId.GOALS]: 10,
     [TileId.WEB_VITALS]: 11,
     [TileId.WEB_VITALS_PATH_BREAKDOWN]: 12,
+
+    // Page Report Tiles
+    [TileId.PAGE_REPORTS_COMBINED_METRICS_CHART]: 13,
```

Comment:
You could start these at 1 if you like, it doesn't really matter as we shouldn't have an active query for both PageReports and the regulaer dashboard at the same time